share/completions/git.fish

```diff
@@ -180,6 +180,22 @@ function __fish_git_files
                     set -ql renamed
                     and set file (string replace -r '\t[^\t]*' '' -- "$line[10..-1]")
                     and set desc $renamed_desc
+                case '2 RM*'
```

Comment:
Can you please add a comment here describing when this porcelain status might be encountered plus the issue number?

Thanks!